src/north/console/Console.class.js

```diff
@@ -17,6 +17,7 @@ class Console extends ApiHandler {
     this.canHandleValues = true
     this.canHandleFiles = true
     this.verbose = applicationParameters.Console.verbose ?? false
+    this.connected = true
```

Comment:
`this.connected = true` should it be instead set if the connect() has been successful in the parent class?

Reply:
I disagree: the parent class is called first to initiate some variables. For some connectors, like MQTT (both for North and South) the connector is connected in async method. So we cannot mark it connected too soon. I know it feels like redondant, but I didn't see another way to manage this case.

Reply:
see branch https://github.com/OptimistikSAS/OIBus/tree/consistent-connect

Reply:
Done, thank you !